db/models/review.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const Sequelize = require('sequelize')
+const db = require('APP/db')
+
+const Review = db.model({
```

Comment:
i think this should be 'db.define' and also have a name string 'reviews' or 'review'.